ros2_modulo_control/Dockerfile

```
@@ -17,6 +21,34 @@ RUN git clone -b galactic --depth 1 https://github.com/ros2/rcl_interfaces.git
 RUN git clone -b galactic --depth 1 https://github.com/ros2/test_interface_files.git
 RUN git clone -b ros2 --depth 1 https://github.com/ros/angles.git
 
+
+FROM ${BASE_IMAGE}:galactic as ros2-control-version-galactic
```

Remember that an ARG is persisted between stages. You will need to re-declare it each time for every stage.
```suggestion
ARG BASE_IMAGE=ghcr.io/aica-technology/ros2-modulo
FROM ${BASE_IMAGE}:galactic as ros2-control-version-galactic
```
It's worth the code duplication to keep the `--local-base` option imo